test/kube2e/README.md

```diff
@@ -1,3 +1,6 @@
+> [!NOTE]
```

Comment:
Thanks for adding this! Below, there is a link to https://github.com/solo-io/gloo/blob/main/devel/testing/kube-e2e-tests.md. Could you update that to reflect the two places where kubernetes e2e tests live?

Reply:
Not sure if I follow- update the link in this README, or update the other markdown file that happens to be referenced below? I'm guessing you mean the second.